Docs/source/usage/parameters.rst

```diff
@@ -1919,9 +1920,42 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Please see the :ref:`data analysis section <dataanalysis-formats>` for more information.
 
 .. _running-cpp-parameters-diagnostics-btd:
+BackTransformed Diagnostics (with support for Plotfile/OpenPMD output)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``BackTransformed`` diag type are used when running a simulation in a boosted frame, to reconstruct output data to the lab frame. This option can be set using ``<diag_name>.diag_type = BackTransformed``. Additional options for this diagnostic include:
 
-Back-Transformed Diagnostics
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* ``<diag_name>.num_snapshots_lab`` (`integer`)
+    Only used when ``<diag_name>.diag_type`` is ``BackTransformed``.
+    The number of lab-frame snapshots that will be written.
+
+* ``<diag_name>.dt_snapshots_lab`` (`float`, in seconds)
+    Only used when ``<diag_name>.diag_type`` is ``BackTransformed``.
+    The time interval inbetween the lab-frame snapshots (where this
+    time interval is expressed in the laboratory frame).
+
+* ``<diag_name>.dz_snapshots_lab`` (`float`, in meters)
+    Only used when ``<diag_name>.diag_type`` is ``BackTransformed``.
+    Distance between the lab-frame snapshots (expressed in the laboratory
+    frame). ``dt_snapshots_lab`` is then computed by
+    ``dt_snapshots_lab = dz_snapshots_lab/c``. Either `dt_snapshots_lab`
+    or `dz_snapshot_lab` is required.
+
+* ``<diag_name>.buffer_size`` (`integer`)
+    Only used when ``<diag_name>.diag_type`` is ``BackTransformed``.
+    The default size of the back transformed diagnostic buffers used to generate lab-frame
+    data is 256. That is, when the multifab with lab-frame data has 256 z-slices,
+    the data will be flushed out. However, if many lab-frame snapshots are required for
+    diagnostics and visualization, the GPU may run out of memory with many large boxes with
+    a size of 256 in the z-direction. This input parameter can then be used to set a
+    smaller buffer-size, preferably multiples of 8, such that, a large number of
+    lab-frame snapshot data can be generated without running out of gpu memory.
+    The downside to using a small buffer size, is that the I/O time may increase due
+    to frequent flushes of the lab-frame data. The other option is to keep the default
+    value for buffer size and use slices to reduce the memory footprint and maintain
+    optimum I/O performance.
+
+Back-Transformed Diagnostics (customized output)
```

Comment:
Customized? Maybe "legacy output option" instead?

Reply:
Either way will work for this. It will be removed from doc soon after this PR is merged.

Reply:
Ok, then let's use this
```suggestion
Back-Transformed Diagnostics (legacy output)
```

Reply:
For follow-up PRs: one thing we need to remember is that we do not remove the old BTD before we have feature parity in the new BTD. That means, we need restart capabilities before we can remove it.

Reply:
Yes. That is the plan. Although I dont believe the old BTD had restart feature, if we are talking about feature parity

Reply:
For clarification, the old BTD did not have restart capabiliy.
So we can actually start removing old BTD. But I still agree that restart has to be the next PR before removing old BTD.